src/resources/i18n/fr.json

```diff
@@ -45,7 +45,13 @@
     "telephone": "Téléphone",
     "fax": "Fax",
     "email": "Adresse e-mail",
-    "currencyCod": "Code de devise"
+    "currencyCod": "Code de devise",
+    "editHospital": "Edit hospital infos",
+    "updateHospital": "Save changes",
+    "saveHospital": "Save",
+    "updated": "Hospital infos updated",
+    "updateSuccess": "Hospital infos have been updated successfully!",
+    "discardChanges": "Changes will be lost. Do you want to discard anyway ?"
```

Comment:
Not translated

Reply:
Please can you translate them?

Reply:
Do you want me to push on this branch?

Reply:
no, you should use Transifex

Reply:
How to do that?

Reply:
I can see this PR has been opened by a bot

Reply:
by operating directly in Transifex, the bot will update this PR with more commits. Open an account on Transifex and ask to join the project (sent invitation to your email anyway)

Reply:
@SilverD3 when you have done, please resolve this conversation